polkadot/node/core/backing/src/lib.rs

```diff
@@ -858,7 +853,7 @@ async fn validate_and_make_available(
 						))
 						.await
 						.map_err(Error::BackgroundValidationMpsc)?;
-					return Ok(())
+					return Ok(());
```

Comment:
please rerun the formatter on other files also (helps during review)

Reply:
Alright

Reply:
Hey @alindima I am having an issue running Clippy. I keep running low on disk space. This is slowing me down a bit. But what do you think so far?

Reply:
I'll have another look soon but generally looks good. I think the node-side part is close to being good to merge